enterprise/frontend/src/metabase-enterprise/auth/index.js

```diff
@@ -69,7 +69,6 @@ PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections =>
 PLUGIN_ADMIN_SETTINGS_UPDATES.push(sections => ({
   ...sections,
   "authentication/saml": {
-    sidebar: false,
```

Comment:
Simplify the logic to handle nested router here https://github.com/metabase/metabase/pull/23571/files#diff-1eff1a3779669ddf20852f1fe3b303e943ef20f201cde2d2f7839190d2687dd7R197-R199

I checked that there are only nested route in authentication and with this change introducing another nested routes in embedding